instrumentation/aws-sdk/aws-sdk-1.11/testing/src/main/java/io/opentelemetry/instrumentation/awssdk/v1_11/AbstractDynamoDbClientTest.java

```diff
@@ -34,13 +41,14 @@ public void sendRequestWithMockedResponse() throws Exception {
 
     server.enqueue(HttpResponse.of(HttpStatus.OK, MediaType.PLAIN_TEXT_UTF_8, ""));
 
+    List<AttributeAssertion> additionalAttributes =
+        Arrays.asList(
+            equalTo(stringKey("aws.table.name"), "sometable"),
+            equalTo(stringKey("db.system"), "dynamodb"),
+            equalTo(stringArrayKey("aws.dynamodb.table_names"), singletonList("sometable")));
```

Comment:
it's a bit confusing to have both `aws.dynamodb.table_names` and `aws.table.name` at the same time, but I don't have a better idea (and `aws.table.name` requires experimental flag to get anyways)

Reply:
Yep I had the same qualm but I don't want to introduce a breaking change but also wanted to align to the sem convs